lib/utils/pyhelp.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of the Micro Python project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Colin Hogben
```

Comment:
> Usually I don't put any copyright header in such cases.
> 
> I believe all files in the repo have the same header template, which has copyright statement, including .h files.

There are some .h files that have no header, eg esp8266/modpyb.h, esp8266/espneopixel.h, esp8266/esppwm.h, stmhal/mphalport.h, extmod/modwebsocket.h.